gtsfm/scene_optimizer.py

```diff
@@ -105,16 +114,22 @@ def create_computation_graph(
         i2Ui1_graph_dict = {}
         v_corr_idxs_graph_dict = {}
 
-        frontend_rot3_errors = []
-        frontend_unit3_errors = []
+        frontend_metrics_dict = {}
 
         for (i1, i2) in image_pair_indices:
             if gt_pose_graph is not None:
                 gt_relative_pose = dask.delayed(lambda x, y: x.between(y))(gt_pose_graph[i2], gt_pose_graph[i1])
             else:
                 gt_relative_pose = None
 
-            (i2Ri1, i2Ui1, v_corr_idxs, rot_error, unit_tran_error,) = self.two_view_estimator.create_computation_graph(
+            (
+                i2Ri1,
+                i2Ui1,
+                v_corr_idxs,
+                rot_error,
+                unit_tran_error,
+                corr_correctness_count,
```

Comment:
not sure if `corr_correctness_count,` is the right variable name -- maybe `correspondence_stats`? or `pair_corr_stats`?

Reply:
I'd go with `correspondence_stats`.